website/integrations/services/knocknoc/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Integrate with Knocknoc
+sidebar_label: Knocknoc
+support_level: community
+---
+
+## What is Knocknoc
+
+> Knocknoc links your single-sign on experience to existing network access controls. It can dynamically orchestrate network access controls (e.g., managing firewall rules in real-time without exposing target machines) or function as an identity-aware gateway.
+>
+> -- https://knocknoc.io/
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `knocknoc.company` is the FQDN of the Knocknoc installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::note
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+## authentik configuration
+
+To support the integration of Knocknoc with authentik, you need to create an application/provider pair and 3 property mappings in authentik.
+
+### Create property mappings in authentik
+
+1. Log in to authentik as an admin, and open the authentik Admin interface.
+2. Navigate to **Customization** -> **Property Mappings** and click **Create** to create a property mapping.
+
+- **Select type**: Select **SAML Provider Property Mapping** as the type and click **Next**.
+
+- **Create SAML Provider Property Mapping**: -**Name**: provide a descriptive name (e.g. `SAML to Knocknoc realName`) -**SAML Attribute Name**: `realName` -**Expression**:
```

Comment:
```suggestion
- **Create SAML Provider Property Mapping**: 
      -**Name**: provide a descriptive name (e.g. `SAML to Knocknoc realName`) 
      -**SAML Attribute Name**: `realName` 
      -**Expression**:
```

Reply:
The formatting was off, needed line breaks, but I might have mucked it up so please review. ;-)